### PR TITLE
[8.0] hackathon fixes

### PR DESCRIPTION
**Pull request by fstagni**
(not worth release notes)

### CODE REVIEW

**Comment by chaen** [comment]
> [fix: fixes for frenglish speakers](https://github.com/DIRACGrid/DIRAC/pull/5940/commits/c91eb1c8e94bc2fcaacab0a81a8b584751794f65)

 :middle_finger:  :laughing:

File: src/DIRAC/WorkloadManagementSystem/DB/JobDB.py [hunk 1]
```diff
@@ -1921,7 +1921,7 @@ def setHeartBeatData(self, jobID, dynamicDataDict):
         # If HeartBeatTime is being set, set it...
         timeStamp = dynamicDataDict.pop("HeartBeatTime", None)
         if timeStamp:
-            req = f"UPDATE Jobs SET HeartBeatTime={timeStamp} WHERE JobID={e_jobID}"
+            req = f"UPDATE Jobs SET HeartBeatTime='{timeStamp}' WHERE JobID={e_jobID}"
```
Comment (chrisburr) [inline src/DIRAC/WorkloadManagementSystem/DB/JobDB.py:1924]: Doesn't this need to be escaped?

File: src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py [hunk 2]
```diff
@@ -817,7 +817,7 @@ def __timeLeft(self):
         self.timeLeft = timeLeft
         if not self.littleTimeLeft:
             if timeLeft and timeLeft < self.grossTimeLeftLimit:
-                self.log.info("TimeLeft bellow %s, now checking with higher frequency" % timeLeft)
+                self.log.info("TimeLeft below %s, now checking with higher frequency" % timeLeft)
```
Comment (chrisburr) [inline src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py:820]:
```suggestion
                self.log.info("Checking with higher frequency as TimeLeft below grossTimeLeftLimit", f"{timeLeft=}")
```

File: src/DIRAC/WorkloadManagementSystem/DB/JobDB.py (hunk 1 above)
Comment (chrisburr) [reply src/DIRAC/WorkloadManagementSystem/DB/JobDB.py:1924]: See #5946

**Review by atsareg — APPROVED** [review]

**Pull request merged**